### ISSUE TITLE
New Comments field in License CPT

### ISSUE THREAD

**Issue by crweiner** [OpenSourceOrg/dotOrg#129]
### Description

Ariel Jolo requested that we create a new comments field within the License CPT. They have created a concept of this on the staging site, but need the comments field to be properly added to the License CPT on production.

They shared that on staging, they added the following in template-parts/content-licenses.php, and would like these styles added to a class instead of being inline:

```html
<div class="entry-content post--content license-content">
  <div>
    <?php the_content(); ?>

    <hr />
    <div style="padding: 2rem; background: #f0E68C">
    <?php 
      if ( osi_field_check( 'comments' ) ) :
        echo wp_kses_post( '<h2 class="wp-block-heading" id="license-comments" style="margin-top:0">' . __( 'Comments', 'osi' ) . '</h2>' );
        osi_the_valid_field( 'comments' );
      endif;
    ?>
    </div>
</div>
```

### Additional context

See ZD link on the internal dev request.

**Comment by nickpagz** [comment]
@ajolo2ky Is this meant to only be comments added via the editor via that field? Or did you need more functionality out of it?

I've implemented this via the repo on staging, and tweaked the styling so there's no background color when the field is empty.

Let me know if this is what you're looking for or you need more.

**Comment by eoinsav** [comment]
From the partner:

> Hi, thanks for the follow up. I tested it and it looks good, please promote it to production when you can.

**Comment by eoinsav** [comment]
Hey @nickpagz! Just want to make sure this didn't get lost during your AFK. We're ready to push your solution live. Thanks!

**Comment by nickpagz** [comment]
Thanks for the nudge @eoinsav 
I've now merged this to production, and also added the "Comments" field in ACF(/SCF?) so it appears in the single license edit pages.

**Comment by eoinsav** [comment]
Nice, thank you!

**Comment by eoinsav** [comment]
Hey @ajolo2ky! This is now live if you would like to close out this issue.

**Comment by ajolo2ky** [comment]
Hi, sorry to reopen this, but we see are facing some issues:

See the comment section on this newly-published page: https://opensource.org/license/osc-license-1-0

there are supposed to be new lines and paragraphs but those are lost. Also, the comment should allow for HTML links to be included.

**Comment by nickpagz** [comment]
Hi @ajolo2ky Looking at the ACF settings, the comment field is set as a Text Area field, which allows HTML markup. I've tested a couple of html tags and they seem to work and display on the front-end. We can also change the field to a WYSIWYG field type if that's better/easier for editing. Though, we would want to move the field out of the "sidebar" area in the editor as the tools get a bit squished. Let me know if that's what you're looking for and I can change it. Thanks!

**Comment by ajolo2ky** [comment]
Thanks @nickpagz let's do that. Let's move it to WYSIWYG if that keeps both the desired structure (allows paragraphs) and HTML.

Agree the sidebar is not the best location, if you can move it to the area below the content, that'd be great.

**Comment by nickpagz** [comment]
@ajolo2ky This is implemented on the staging site. 
Note that since the field location is controlled by the group settings, I had to create a new field group "License Data - Comments" and move the "Comments" field over to the new field group to get it to work. Have a look and let me know what you think before I implement it on the production site.

**Comment by ajolo2ky** [comment]
For some reason I can't use the WYSIWYG editor in Visual mode. I have to go to Text/Code to be able to add content there.

**Comment by nickpagz** [comment]
@ajolo2ky I was able to replicate this about half of the time I tested, and it almost always happens on a new license post. This seems to be a common enough issue with ACF they've added an option to activate the editor when clicked. I've enabled the option and it seems to be working consistently now. Let me know.

**Comment by ajolo2ky** [comment]
Yes, this is working now. Thank you very much, please promote to production when convenient.

**Comment by ajolo2ky** [comment]
Hi @nickpagz please let me know when you get a chance to promote this to production. Thanks.

**Comment by nickpagz** [comment]
@ajolo2ky I've added this to production and seems to be working. Have a look and let me know if you notice any issues.
